common/translations/lang/fr.json

```diff
@@ -327,338 +286,380 @@
-    "HELP_SEC_DESC_8": "Si cet outil ne vous plaît pas, alors ne l'utilisez surtout pas. Nous l'avons créé pour qu'il aide les gens à générer des portefeuilles et à effectuer des transactions sans avoir besoin de plonger dans les lignes de commandes ni faire tourner un nœud complet. À nouveau, n'hésitez pas à nous faire part de vos doutes et nous répondrons aussi rapidement que possible. Merci ! ",
-    "HELP_FAQ_TITLE": "Plus de réponses utiles aux questions fréquentes ",
-    "HELP_CONTACT_TITLE": "Moyens de nous contacter",
-    "ONBOARD_WELCOME_TITLE": "Welcome to MyCrypto.com",
-    "ONBOARD_WELCOME_CONTENT__1": "Please take some time to understand this for your own safety. 🙏",
-    "ONBOARD_WELCOME_CONTENT__2": "Your funds will be stolen if you do not heed these warnings.",
-    "ONBOARD_WELCOME_CONTENT__3": "We know this click-through stuff is annoying. We are sorry.",
-    "ONBOARD_WELCOME_CONTENT__4": "What is MyCrypto? ",
-    "ONBOARD_WELCOME_CONTENT__5": "MyCrypto is a free, open-source, client-side interface.",
-    "ONBOARD_WELCOME_CONTENT__6": "We allow you to interact directly with the blockchain while remaining in full control of your keys &amp; your funds.",
-    "ONBOARD_WELCOME_CONTENT__7": "**You** and **only you** are responsible for your security.",
-    "ONBOARD_WELCOME_CONTENT__8": "We cannot recover your funds or freeze your account if you visit a phishing site or lose your private key.",
-    "ONBOARD_BANK_TITLE": "MyCrypto is not a Bank",
-    "ONBOARD_BANK_CONTENT__1": "When you open an account with a bank or exchange, they create an account for you in their system.",
-    "ONBOARD_BANK_CONTENT__2": "The bank keeps track of your personal information, account passwords, balances, transactions and ultimately your money.",
-    "ONBOARD_BANK_CONTENT__3": "The bank charge fees to manage your account and provide services, like refunding transactions when your card gets stolen.",
-    "ONBOARD_BANK_CONTENT__4": "The bank allows you to write a check or charge your debit card to send money, go online to check your balance, reset your password, and get a new debit card if you lose it.",
-    "ONBOARD_BANK_CONTENT__5": "You have an account *with the bank or exchange* and they decide how much money you can send, where you can send it, and how long to hold on a suspicious deposit. All for a fee.",
+    "ONBOARD_WELCOME_TITLE": "Bienvenue sur MyCrypto.com",
+    "ONBOARD_WELCOME_CONTENT__1": "Veuillez prendre le temps de lire ceci pour votre propre sécurité. 🙏",
+    "ONBOARD_WELCOME_CONTENT__2": "Vos fonds seront volés si vous ne tenez pas compte de ces avertissements.",
+    "ONBOARD_WELCOME_CONTENT__3": "Nous sommes conscients que cette étape est longue. Nous vous prions de nous en excuser.",
+    "ONBOARD_WELCOME_CONTENT__4": "Qu'est-ce que MyCrypto ? ",
+    "ONBOARD_WELCOME_CONTENT__5": "MyCrypto est une interface coté client, open-source et gratuite.",
```

Comment:
open-source => code source ouvert/libre

Reply:
"open-source" is widely used in French, but we can indeed change it to "MyCrypto est une interface coté client, gratuite et dont le code source est libre."